docs/source/theory.rst

```diff
@@ -272,9 +276,75 @@ Finally, convective heat fluxes can be applied to boundaries:
 
 where :math:`h` is the heat transfer coefficient and :math:`T_{\mathrm{ext}}` is the external temperature.
 
----------------
+Kinetic surface model
+^^^^^^^^^^^^^^^^^^^^^
+
+Modelling hydrogen retention or outgassing might require considering the kinetics of surface processes. 
+A representative example is the hydrogen uptake from a gas phase, when the energy of incident atoms/molecules is not high enough to 
+overcome the surface barrier for implantation. The general approach to account for surface kinetics :cite:`Pick1985, Hodille2017, Guterl2019, Schmid2021` consists in 
+introducing hydrogen surface species :math:`c_\mathrm{s}\,[\mathrm{m}^{-2}]`. 
+
+Evolution of hydrogen surface concentration is determined by the atomic flux balance at the surface, as sketched in the simplified energy diagram below.
+
+.. thumbnail:: images/potential_diagram.png
+    :align: center
+    :width: 800
+
+    Idealised potential energy diagram for hydrogen near a surface of an endothermic metal. Energy levels are measured from the :math:`\mathrm{H}_2` state
+
+The governing equation for surface species is:
+
+.. math::
+    :label: eq_surf_conc
+    
+    \dfrac{d c_\mathrm{s}}{d t} = J_\mathrm{bs} - J_\mathrm{sb} + J_\mathrm{vs}~\text{on}~\delta\Omega
+
+where :math:`J_\mathrm{bs}\,[\mathrm{m}^{-2}\,\mathrm{s}^{-1}]` is the flux of hydrogen atoms from the subsurface (bulk region just beneath the surface) onto the surface, 
+:math:`J_\mathrm{sb}\,[\mathrm{m}^{-2}\,\mathrm{s}^{-1}]` is the flux of hydrogen atoms from the surface into the subsurface, and :math:`J_\mathrm{vs}\,[\mathrm{m}^{-2}\,\mathrm{s}^{-1}]` is the net flux of hydrogen 
```

Comment:
does J_vs contains the desorbing flux and all abstraction fluxes (ELy Rideal recombination etc) ?

Reply:
In general, yes, but it should be defined by a user. Here is an example from [one](https://github.com/KulaginVladimir/FESTIM-SurfaceKinetics-Validation/blob/main/D_damagedW/D_damagedW.ipynb) validation case on how it can be defined:
```python
def J_vs(T, surf_conc, t):
    phi_atom = SP * Gamma_atom * (1 - surf_conc / n_surf) # absorption
    phi_exc = Gamma_atom * sigma_exc * surf_conc # abstraction
    phi_des = 2 * nu0 * (lambda_des * surf_conc) ** 2 * f.exp(-2 * E_des / F.k_B / T) # desorption
    return phi_atom - phi_exc - phi_des
```

Reply:
Maybe this could be explicited then? something like

```
is the net flux of hydrogen atoms from the vacuum onto the surface (eg recombination, dissociation, Eley-Rideal recombination, etc.).
```

Reply:
In this case, should we somehow clarify that the flux comes **onto the surface**: `J_vs = +Dissoc. - Recomb. - Eley-Rideal recomb ...`?

Reply:
Probably

Reply:
What about this:

Assuming that there are $N$ processes leading to adsorption of hydrogen with flux $J_{\mathrm{ads},i}$ (e.g. dissociation from the gase phase, adsorption of low-energy ions, etc) and $M$ processes leading to desorption of hydrogen with flux $J_{\mathrm{des},j}$ (e.g. molecular recombindation, Eley-Rideal recombination, etc), the resulting flux onto the surface is: $J_\mathrm{vs}=\sum\limits_i^N J_{\mathrm{ads},i}-\sum\limits_j^M J_{\mathrm{des},j}$

Reply:
Maybe just say J_vs = J_in - J_out where J_in is the sum of all fluxes coming from the vacuum to the surface and J_out from the surface to the vacuum

Or something like that and then give a few examples

Reply:
A few examples on different processes that can be included?

Reply:
Yes just like you had in your previous comment

Reply:
I updated this part. Let me know what you think.

Reply:
Looks good to me!